reactjs-films-homework/src/index.tsx

```diff
@@ -2,10 +2,14 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.scss';
```

Comment:
please order the imports
The convention could be:
imports of  libs
relative imports from project
imports of styles